apprentice_teacher.rb

```diff
@@ -1,51 +1,28 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative'module_staff'
+require_relative'high_give_module.rb'
+require_relative'school'
+
+class ApprenticeTeacher < School
```

Comment:
@salogel42 Thank you all the feedback. I really appreciate it . I'll refactor it tonight.